WalletWasabi.Fluent/ViewModels/Wallets/Home/Tiles/WalletPieChartTileViewModel.cs

```diff
@@ -20,24 +24,44 @@ public DataLegend(Money amount, string label, string hexColor, double percentSha
 		public double PercentShare { get; }
 	}
 
-	public partial class WalletPieChartTileViewModel : ViewModelBase
+	public partial class WalletPieChartTileViewModel : TileViewModel
 	{
-		[AutoNotify] private IList<(string color, double percentShare)> _testDataPoints;
-		[AutoNotify] private IList<DataLegend> _testDataPointsLegend;
+		private readonly Wallet _wallet;
+		private readonly Config _config;
 
-		public WalletPieChartTileViewModel(Wallet wallet)
+		[AutoNotify] private IList<(string color, double percentShare)>? _testDataPoints;
+		[AutoNotify] private IList<DataLegend>? _testDataPointsLegend;
+
+		public WalletPieChartTileViewModel(Wallet wallet, Config config, IObservable<Unit> balanceChanged)
+		{
+			_wallet = wallet;
+			_config = config;
+
+			balanceChanged.Subscribe(_ => Update());
+
+			Update();
+		}
+
+		private void Update()
 		{
+			var privateCoins = _wallet.Coins.FilterBy(x => x.HdPubKey.AnonymitySet > _config.PrivacyLevelStrong);
+			var normalCoins = _wallet.Coins.FilterBy(x => x.HdPubKey.AnonymitySet < _config.PrivacyLevelStrong);
```

Comment:
In the `PrivacyControlViewModel` we use the `_wallet.ServiceConfiguration.GetMixUntilAnonymitySetValue()` as a limit to decide whether a coin is private or not. I think this should be used here too. And the `config` can be completely removed.

Reply:
done.